### PR TITLE
1303 add gold tooltip

### LINKED ISSUE #1303: add tooltip for `GOLD Ecosystems`
Help text should be populated with information from the schema tooltip annotation for [gold_path_field](https://microbiomedata.github.io/nmdc-schema/gold_path_field/)

This is in nmdc-schema main, there should be a tagged release tomorrow.

The two formats these tooltip annotations could take are described in 
https://github.com/microbiomedata/nmdc-server/issues/750

### PR DESCRIPTION
**Pull request by marySalvi**
closes #1303 
- Adds tooltip for GOLD Ecosystems
- Also converts all schema JSON references to yaml

### CODE REVIEW

File: web/src/components/MenuContent.vue
Comment (naglepuff) [inline web/src/components/MenuContent.vue:1]: I don't see facet definitions anymore. Is this expected? Do these need to be populated in the schema again?

Prod:
image

This branch:
image

**Review by naglepuff — CHANGES_REQUESTED** [review]

File: web/src/components/MenuContent.vue
Comment (marySalvi) [reply web/src/components/MenuContent.vue:1]: Good catch, I'll look into this

**Comment by aclum** [comment]
@marySalvi  how are things coming along with the change requests? Will we be able to include this feature in the monthly release scheduled for Monday?

**Comment by marySalvi** [comment]
> @marySalvi how are things coming along with the change requests? Will we be able to include this feature in the monthly release scheduled for Monday?

@aclum Yes, this should be able to be included.

File: web/src/components/MenuContent.vue
Comment (marySalvi) [reply web/src/components/MenuContent.vue:1]: fixed in 27d656faf1c44922354cf864f984d20f4b5febd2

File: web/src/data/api.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { merge } from 'lodash';
 import axios, { AxiosError } from 'axios';
 import { setupCache } from 'axios-cache-adapter';
-import NmdcSchema from 'nmdc-schema/nmdc_schema/nmdc.schema.json';
+// @ts-ignore
+import nmdc from 'nmdc-schema/nmdc_schema/nmdc_materialized_patterns.yaml';
```
[the remark this answers is not in the document]
Comment (marySalvi) [reply web/src/data/api.ts]: changed in 1009546afe4126182a42f76c1866e5c267fdf0fa

**Review by naglepuff — APPROVED** [review]
Great! I see the text in the sidebar flyouts again.

**Pull request merged**